### PR TITLE
include alby extension

### LINKED ISSUE #1296: Link to Alby How To "Connect to BTCPay Server"
Alby has a guide on how to [Connect to BTCPay Server](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-lightning-account/connect-your-alby-lightning-account-to-other-apps/connect-to-btcpay-server) which we can link to.

### PR DESCRIPTION
**Pull request by TChukwuleta**
Resolves #1296   by including Alby to BTCPay docs

### CODE REVIEW

File: docs/LightningNetwork.md [hunk 1]
```diff
@@ -183,3 +182,7 @@ cd btcpayserver-docker
 Once your on-chain lightning node is funded, it's time to connect to other nodes on the network and open payment channels.
 \
 Check out [Payment channels](./LightningNetwork_PaymentChannels.md) for recommendations on opening payment channels, liquidity management and more.
+
+## Alby Extension
+
+[Alby](https://getalby.com/) is a free, fast, simple, and convenient way to send and receive Bitcoin payments with your normal browser on the Bitcoin Lightning Network with ease. You can now connect your BTCPay wallet directly to your Alby extension. To learn more on how to connect your BTCPay wallet to alby, visit [here](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-lightning-account/connect-your-alby-lightning-account-to-other-apps/connect-to-btcpay-server)
```
Comment (dennisreimann) [inline docs/LightningNetwork.md:188]:
```suggestion
[Alby](https://getalby.com/) is a free, fast, simple, and convenient way to send and receive Bitcoin payments with your normal browser on the Bitcoin Lightning Network with ease. You can now connect your BTCPay wallet directly to your Alby account. To learn more on [how to connect your BTCPay wallet to Alby](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-lightning-account/connect-your-alby-lightning-account-to-other-apps/connect-to-btcpay-server).
```

Comment (TChukwuleta) [reply docs/LightningNetwork.md]: updated

File: docs/LightningNetwork.md [hunk 2]
```diff
@@ -183,3 +182,7 @@ cd btcpayserver-docker
 Once your on-chain lightning node is funded, it's time to connect to other nodes on the network and open payment channels.
 \
 Check out [Payment channels](./LightningNetwork_PaymentChannels.md) for recommendations on opening payment channels, liquidity management and more.
+
+## Alby Extension
+
+[Alby](https://getalby.com/) is a free, fast, simple, and convenient way to send and receive Bitcoin payments with your normal browser on the Bitcoin Lightning Network with ease. You can now connect your BTCPay wallet directly to your Alby account. To learn more on [how to connect your BTCPay wallet to Alby](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-lightning-account/connect-your-alby-lightning-account-to-other-apps/connect-to-btcpay-server).
```
Comment (dennisreimann) [inline docs/LightningNetwork.md:188]: Sorry, I missed one word here:

```suggestion
[Alby](https://getalby.com/) is a free, fast, simple, and convenient way to send and receive Bitcoin payments with your normal browser on the Bitcoin Lightning Network with ease. You can now connect your BTCPay wallet directly to your Alby account. Learn more on [how to connect your BTCPay wallet to Alby](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-lightning-account/connect-your-alby-lightning-account-to-other-apps/connect-to-btcpay-server).
```

File: docs/LightningNetwork.md [hunk 3]
```diff
@@ -6,11 +6,10 @@ After deploying BTCPay Server, you may want to experiment with an innovative sec
 
 This guide will show you how to set up your Lightning Network (LN) node in BTCPay Server and guide you through the basics.
 
-BTCPay Server currently offers three implementations of the Lightning Network:
+BTCPay Server currently offers two implementations of the Lightning Network:
 
 - [LND](https://github.com/lightningnetwork/lnd)
 - [Core Lightning (CLN)](https://github.com/ElementsProject/lightning) (formerly c-lightning)
-- [Eclair](https://github.com/ACINQ/eclair)
```
[the remark this answers is not in the document]
Comment (TChukwuleta) [reply docs/LightningNetwork.md:13]: @dennisreimann mentioned that BTCPay doesn't support Eclair

[the remark this answers is not in the document]
Comment (dennisreimann) [reply docs/LightningNetwork.md:13]: Yes, sorry, that was a mistake on my end: I thought we removed it at some point.

File: docs/LightningNetwork.md [hunk 4]
```diff
@@ -6,11 +6,10 @@ After deploying BTCPay Server, you may want to experiment with an innovative sec
 
 This guide will show you how to set up your Lightning Network (LN) node in BTCPay Server and guide you through the basics.
 
-BTCPay Server currently offers three implementations of the Lightning Network:
+BTCPay Server currently offers two implementations of the Lightning Network:
 
 - [LND](https://github.com/lightningnetwork/lnd)
 - [Core Lightning (CLN)](https://github.com/ElementsProject/lightning) (formerly c-lightning)
```
Comment (dennisreimann) [inline docs/LightningNetwork.md:12]:
```suggestion
- [Core Lightning (CLN)](https://github.com/ElementsProject/lightning) (formerly c-lightning)
- [Eclair](https://github.com/ACINQ/eclair)
```

File: docs/LightningNetwork.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ After deploying BTCPay Server, you may want to experiment with an innovative sec
 
 This guide will show you how to set up your Lightning Network (LN) node in BTCPay Server and guide you through the basics.
 
-BTCPay Server currently offers three implementations of the Lightning Network:
+BTCPay Server currently offers two implementations of the Lightning Network:
```
Comment (dennisreimann) [inline docs/LightningNetwork.md:9]:
```suggestion
BTCPay Server currently offers these implementations of the Lightning Network:
```

File: docs/LightningNetwork.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TChukwuleta) [reply docs/LightningNetwork.md]: I'd add it back then

**Review by pavlenex — APPROVED** [review]

**Pull request merged**